### ISSUE TITLE
JAR file does nothing

### ISSUE THREAD

**Issue by mcao9** [Expugn/S-argo#7]
Please review the info for Bug Reports in CONTRIBUTING.md before submitting
your issue. Thank you!

### Bot Version
v2.5

### Type of Issue
The JAR file does not do anything and the S'argo bot fails to come online.

### Bug Summary
If I double click the JAR file, a log folder is created but that's it. I've also seen the previous post on this issue and created the run.bat file to debug this problem. The command prompt opens briefly before closing but I've managed to read something about "FAILED TO READ BANNER FILE" and DiscordException

### How to Reproduce the Issue
Happens on step 5 of Getting Started.

**Comment by Expugn** [comment]
to be sure, you have the `Settings.xml` with all the required information (bot token, your discord ID, etc) in the appropriate location (in the `/data/` folder), correct?

**Comment by mcao9** [comment]
Yes, I pasted them in.

**Comment by Expugn** [comment]
did you change the link between `<GitHubDataRepository></GitHubDataRepository>` in the `Settings.xml` by accident?

**Comment by mcao9** [comment]
Is it this? 
https://raw.githubusercontent.com/Expugn/S-argo_Data_v2/master/
I don't think I touched anything except the Bot token and user id.

**Comment by Expugn** [comment]
yep that's correct. can you walk me through on how you set up the bot?

**Comment by mcao9** [comment]
So I downloaded both the files (v.2.5) and put them in a folder
I created a application for the bot.
Pasted the client ID in the link provided.
Authorized the Bot with all permissions
Edited the Settings.xml by replacing the BOT Token with mine
Right clicked my discord profile and pressed "Copy ID" 
Pasted that in Bot Owner ID
Saved it and renamed it "Settings.xml"
OMGosh just figured it out. I renamed it Settings.xml so the file was actually named Settings.xml.xml
